src/astro/databases/google/bigquery.py

```diff
@@ -236,3 +256,189 @@ def gs_to_bigquery(
         self.hook.insert_job(
             configuration=job_config,
         )
+
+    def load_s3_file_to_bigquery(
+        self,
+        source_file: File,
+        target_table: Table,
+        if_exists: LoadExistStrategy = "replace",
+        native_support_kwargs: Optional[Dict] = None,
+        **kwargs,
+    ):
+        """
+         Load content of multiple files in S3 to output_table in Bigquery by using a datatransfer job
+        Note - To use this function we need
+            1. Enable API on Bigquery
+            2. Enable Data transfer service on Bigquery, which is a chargeable service
+        for more information refer - https://cloud.google.com/bigquery-transfer/docs/enable-transfer-service
+
+        :param source_file: Source file that is used as source of data
+        :param target_table: Table that will be created on the bigquery
+        :param if_exists: Overwrite table if exists. Default 'replace'
+        :param native_support_kwargs: kwargs to be used by method involved in native support flow
+        """
+        native_support_kwargs = native_support_kwargs or {}
+
+        project_id = self.get_project_id(target_table)
+
+        if if_exists == "replace":
+            # We need to create an empty table as datatransfer job can only append to an existing table,
+            # so we need to create a table.
+            self.create_empty_table(source_file, target_table)
```

Comment:
We should also support the case where the user gives the Table schema using `Table(columns=[...])`. I'm okay with this being addressed in a separate PR if we think it is out of the scope. This will give us feature parity with Snowflake.

Reply:
@tatiana I'll address this in a separate PR. Created a ticket for the same - https://github.com/astronomer/astro-sdk/issues/532